labtool.py

```diff
@@ -6,13 +6,13 @@
 
 MAX_DEVICE_COUNT = 32
 MAX_THREADS = 16
-docker_image = "snapos/flex:latest"
+docker_image = "snaproute/dockerlab:flexswitch" #"snapos/flex:latest"
```

Comment:
It doesn't appear that there is a public 'snaproute' repository on dockerhub yet.  Let's wait to publish this change until that's available